src/main/java/core/basesyntax/strategy/transactions/TransactionHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.transactions;
+
+public interface TransactionHandler {
+    void produceTransaction(FruitTransaction fruitTransactionImpl);
+}

```

Comment:
why this method was called `produceTransaction`?
let's use `process`, `handle` or `apply`